apiary.apib

```diff
@@ -2601,6 +2602,82 @@ enabled. See <a href="/#reference/0/users/create-two-factor-authentication-secre
             "Message": ""
         }
 
+### Login with token [POST /organizations/auth/token]
+
+Login with a user token
+
+**NOTE:** User token obtain from this endpoint does not allow you to take 
```

Comment:
@sambhare Is this NOTE valid? As far as I can see, the returned user token should be able to take measurements. And we might want to add more description instead of just a NOTE.

Reply:
Can you test if you can measure with a user token obtained via
/users/login vs
/organizations/auth/token

Reply:
User tokens obtained from /user/login and /organizations/auth/ can both be passed to EP /organizations/auth/token, and the response is the same user token as the requested one. 

So the user token obtained from /user/login and the user token obtained from /organizations/auth/token with request body of token which is obtained from /user/login, both of them are able to create measurements. But the user token obtained from /organizations/auth/token with request body of token which is obtained from /organizations/auth/, then it can't create a measurement.